homeassistant/components/maico/const.py

```diff
@@ -0,0 +1,9 @@
+"""Constants for the Maico integration."""
+
+from __future__ import annotations
+
+import httpx
+
+DOMAIN = "maico"
+
+CONNECTION_ERRORS = (KeyError, httpx.HTTPError)
```

Comment:
I haven't read forward to see where this is used yet, but it does strike me as very odd already that a KeyError is a connection error.